### ISSUE TITLE
Cura Signing In

### ISSUE THREAD

**Issue by lion7718** [Ultimaker/Cura#19549]
### Cura Version

5.8.0

### Operating System

Windows 10

### Printer

SV06, SV06 Plus, Ender 3

### Reproduction steps

Open Cura, Cura I'm not signed in. Click Sign in, browser opens I login & now I'm signed into Cura.

### Actual results

I'm signed into Cura

### Expected results

I close Cura for a few hours & reopen & I'm not signed in & have to do the process all over again.

This issue appeared last year & has not been fixed.

### Add your .zip and screenshots here ⬇️

![Capture](https://github.com/user-attachments/assets/a49586aa-7df5-4cfd-9038-f1de25c2fc24)

**Comment by GregValiant** [comment]
I've never had to log in and I have numerous Cura versions installed.  You might have created a situation when you installed Cura that now requires a log-in every time?

Maybe @Asterchades can help with this.

**Comment by lion7718** [comment]
Exactly, I have to sign in every time. It's annoying. It happened during one of the Cura upgrades.

**Comment by Asterchades** [comment]
I can't say that I've ever had to sign into Cura. And I mean *ever* - even using the marketplace hasn't ever required it. I'm honestly not even sure I *have* a login.

To my knowledge (and there's a great big hole here), the only two reasons you might need to sign in would be to back up your settings remotely to the Ultimaker servers, or for the Enterprise edition of Cura (which also requires a subscription). The former is dealt with by a plug-in called **Cura Backups** that defaults to disabled (though if you've enabled it, that will persist through new installations), so perhaps that's something to look for in case you've configured it to run automatically - it's the fifth option in the "Bundled Plugins" section.

_If you still wish to perform local backups you can manually copy the files you find in the **Show configuration folder** help menu to achieve the same result._

**Comment by lion7718** [comment]
I do have Cura Backups installed, so I should disable it..??

**Comment by Asterchades** [comment]
As a diagnostics step, yes. If it doesn't help you can always re-enable it afterwards.

If it stops asking you for a login afterwards at least you have a culprit. At that point maybe you can check the settings to see if there's some kind of facility to save the login details, or just leave it disabled and back up your settings manually. There is, of course, always the possibility that something was broken with it at some point last year, but that will have to be addressed by someone more familiar with it than I am.

**Comment by lion7718** [comment]
I just disabled it and restated Cura, I got this message.


![Capture](https://github.com/user-attachments/assets/cf6cf93b-ec27-46d1-b60f-970d20c99e4a)

**Comment by Asterchades** [comment]
Well, it's _different_ - even if not outright better. So that's something, I guess.

I feel like at this point a log file is going to be essential - hopefully it contains details as to when it's trying to log in, even if not explicitly why it's trying. You can find the log file in the directory opened by the **Show Configuration Folder** option in the Help menu. I feel like the one with Backups enabled would be more beneficial, but if you're feeling keen then both may prove useful.

**Comment by lion7718** [comment]
I don't see the log file.

![Capture](https://github.com/user-attachments/assets/101bd1d8-a3a5-41da-a751-e7f91a8ac404)

**Comment by Asterchades** [comment]
You have extensions hidden (a stupid default setting on MS' behalf and major security risk, IMO). It should be the second file called "cura" with the lines across the icon.

**Comment by lion7718** [comment]
I should have known better, didn't think of that..lol

[cura.log](https://github.com/user-attachments/files/16648388/cura.log)

**Comment by Asterchades** [comment]
If I'm reading this correctly, it looks like it's connecting to the cloud services for an Ultimaker 3 printer - one that's not on your list. So I can only assume that, at some point, it has been in your list (perhaps due to a project file you've loaded) and it's enabled the corresponding services for operating it.

In which case, disabling the associated plugins _may_ stop it asking you to log in. If I compare your log to mine (which does have them disabled), I would be looking to disable the **Ultimaker Machine Actions** and **UM3 Network Printing** plugin from the Bundled Plugin section (they should be a little above the plethora of "Version Upgrade" plugins).

_Of course, why it isn't correctly preserving the login session may well be a bug. Or it could be some sort of isolated weirdness by virtue of having the plugins enabled without a corresponding printer configured, then migrating across multiple Cura versions._

**Comment by lion7718** [comment]
Well, it looks like that was the issue (so far anyway).
I've opened & closed Cura 3 times & now I'm signed in.
I can't tell you how annoying this has been. I would Google the problem from time to time & never saw an answer.
I assumed that UM decided because I didn't use one of their printers I had to sign in every time.
Thanks again for your help, it is highly appreciated.

**Comment by Asterchades** [comment]
No worries. I'm going to close this one up for now, but if it comes up again feel free to write back (it will ping both myself and Greg) and it can be opened again if needs be.

**Comment by lion7718** [comment]
Well, I spoke too soon. I just opened Cura & I was signed out & had to sign back in again.

**Comment by lion7718** [comment]
Here is a new Cura.log with the extensions you had me disable.
[cura.log](https://github.com/user-attachments/files/16648899/cura.log)

**Comment by Asterchades** [comment]
For one last experiment, try disabling Cura Backups in addition to the other two - and maybe the Firmware Update Checker for good measure (it shouldn't be necessary, but I question what it will do other than take time to load without an Ultimaker printer).

Failing that, I'll open this back up and fire off a ping to someone more familiar with the inner workings of Cura. It could well be that there's a bug here I'm just not capable of seeing, but I'd rather not bother them until the DiY options (that I know of) have been exhausted *without going to the extent of clearing all of your settings.*

**Comment by lion7718** [comment]
Ok, I just disabled Cura Backups 7 Firmware Update Checker.
I will post back later to tell you the results.
Here is the newest log file.

[cura.log](https://github.com/user-attachments/files/16649176/cura.log)

**Comment by lion7718** [comment]
No, still doesn't work.

![Capture](https://github.com/user-attachments/assets/c11da58d-601d-4fd7-83c3-d3a15b379058)

**Comment by Asterchades** [comment]
Dang it. Anything more from me at this point just feels like it's going to be random guesses, and we could be here for a while trying to find a solution if that's the best I can do.

@nallath I think you might need to take a look at this one.

**Comment by lion7718** [comment]
I just installed Cura 5.9 and the issue is still present.

**Comment by lion7718** [comment]
I know this was marked for someone else, but I'm still having this issue.

**Comment by fieldOfView** [comment]
Clone Cura, then open cura.cfg in the configuration folder (that's the other 'cura', the one with the gear in it) in a text editor such as notepad, and under `[general]` remove the line that starts with `ultimaker_auth_data = `. This should make Cura forget that you were ever logged in to begin with, so it will not try to log in when you start Cura and this will not fail.

**Comment by lion7718** [comment]
**remove the line that starts with ultimaker_auth_data =**

This line does not exist.

[cura.zip](https://github.com/user-attachments/files/20125486/cura.zip)
